arwen/host/execution.go

```diff
@@ -209,7 +215,12 @@ func (host *vmHost) CreateNewContract(input *vmcommon.ContractCreateInput) ([]by
 
 	err = runtime.CreateWasmerInstance(input.ContractCode, gasForDeployment)
 	if err != nil {
-		fmt.Println("arwen Error: ", err.Error())
+		output.Transfer(input.CallerAddr, address, 0, input.CallValue, nil)
```

Comment:
we can take this code out output.Transfer in case there is an error. If the any vmexecution returns with error, only the error code and the returnData is taken into consideration by the smart contract processor.

Reply:
Fixed, we do not undo the transfer anymore - since the transfer will not take place anyway. Correct?

Reply:
correct. it is not saved in the vmoutput.